api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/TxRTTPLHIVDateset.java

```diff
@@ -0,0 +1,43 @@
+package org.openmrs.module.eptsreports.reporting.library.datasets;
+
+import org.openmrs.module.eptsreports.reporting.library.cohorts.TxRttCohortQueries;
+import org.openmrs.module.eptsreports.reporting.library.indicators.EptsGeneralIndicator;
+import org.openmrs.module.eptsreports.reporting.utils.EptsReportUtils;
+import org.openmrs.module.reporting.dataset.definition.CohortIndicatorDataSetDefinition;
+import org.openmrs.module.reporting.dataset.definition.DataSetDefinition;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TxRTTPLHIVDateset extends BaseDataSet {
```

Comment:
@cfaife you made fixes to other datasets in this code base?

Reply:
sorry @ningosi  i did not get  your point

Reply:
I meant is this dataset **TxRTTPLHIVDateset** related to quality improvement report? maybe you fixed an issue there in this commit

Reply:
pardon @ningosi  im  not following you we 5  file affected in this  PR
![image](https://user-images.githubusercontent.com/18709463/99964957-d06ec300-2d9c-11eb-9dac-5233c583ad3e.png)
And none of them are co-related with  TxRTTPLHIVDateset.
Did you see  the `files changes` tab?